tests/snuba/api/endpoints/test_organization_events_mep.py

```diff
@@ -3247,6 +3250,42 @@ def test_transaction_user_misery(self) -> None:
             },
         }
 
+    def test_support_columns_from_tables(self) -> None:
```

Comment:
I wonder if this test is useful at all since it's the task that was not creating the rows as enabled.
In other words, removing the changes in `src` for this PR does not change the outcome of this test.

Reply:
We should just be on the safe side and check querying and creation for all changes moving forward.